x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/types.ts

```diff
@@ -65,10 +65,13 @@ export interface SearchField {
   weight: number;
 }
 
-export interface SearchSettings {
+export interface SearchSettingsRequest {
```

Comment:
Create a different interface for requests, that does not contain `precision_enabled`